pytorch_lightning/trainer/connectors/env_vars_connector.py

```diff
@@ -18,27 +18,25 @@
 from pytorch_lightning.utilities.argparse import get_init_arguments_and_types, parse_env_variables
 
 
-def overwrite_by_env_vars(fn: Callable) -> Callable:
+def defaults_from_env_vars(fn: Callable) -> Callable:
```

Comment:
Do you want to use this occasion and make the function protected?

```suggestion
def _defaults_from_env_vars(fn: Callable) -> Callable:
```